opm/simulators/wells/PerforationData.hpp

```diff
@@ -35,6 +35,14 @@ struct PerforationData
     std::size_t ecl_index;
 };
 
+struct PerforationRates
+{
+    double dis_gas{};
```

Comment:
Will this zero-initialize? Even if it does, would it not be preferable to write it as `member = 0.0` to make it clearer?

Reply:
it does zero initialize but sure, i can use more familiar syntax.

Reply:
As @akva2 correctly states, [list initialisation](https://en.cppreference.com/w/cpp/language/list_initialization) will perform [value initialisation](https://en.cppreference.com/w/cpp/language/value_initialization) in this case which, for built-in arithmetic types, will assign the type's zero value.